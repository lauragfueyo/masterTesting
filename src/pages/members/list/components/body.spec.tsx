import * as React from 'react';
import { shallow } from 'enzyme';
import { Body } from './body';
import { Member } from '../viewModel';

describe('pages/members/list/components/body tests', () => {
    it('should render as expected when passing required properties', () => {
        // Arrange
        const props: Member[] = [
            {
                id: 1,
                name: 'test name',
                avatarUrl: 'test url',
            },
        ];
  
        // Act
        const component = shallow(
            <Body members={props} />
        );
  
        // Assert
        expect(component).toMatchSnapshot();
   });
});