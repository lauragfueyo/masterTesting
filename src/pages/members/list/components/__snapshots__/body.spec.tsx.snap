// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`pages/members/list/components/body tests should render as expected when passing required properties 1`] = `
<tbody>
  <Component
    key="1"
    member={
      Object {
        "avatarUrl": "test url",
        "id": 1,
        "name": "test name",
      }
    }
  />
</tbody>
`;
